apps/sasquatch/src/main/res/values/settings.xml

```diff
@@ -67,6 +67,15 @@
     <string name="appcenter_distribute_debug_state_title" tools:ignore="MissingTranslation">Distribute in debug state</string>
     <string name="appcenter_distribute_debug_summary_enabled" tools:ignore="MissingTranslation">Distribute in debug is enabled</string>
     <string name="appcenter_distribute_debug_summary_disabled" tools:ignore="MissingTranslation">Distribute in debug is disabled</string>
+    <string name="appcenter_distribute_track_state_key" tools:ignore="MissingTranslation">appcenter_distribute_track_state_key</string>
+    <string name="appcenter_distribute_track_state_title" tools:ignore="MissingTranslation">Update track</string>
+    <string name="appcenter_distribute_track_public_enabled" tools:ignore="MissingTranslation">Distribute uses public track</string>
+    <string name="appcenter_distribute_track_private_enabled" tools:ignore="MissingTranslation">Distribute uses private track</string>
+    <string name="appcenter_distribute_update_track_before_start_key" tools:ignore="MissingTranslation">appcenter_distribute_update_track_before_start_key</string>
+    <string name="appcenter_distribute_update_track_before_start_value" tools:ignore="MissingTranslation">appcenter_distribute_update_track_before_start_value</string>
+    <string name="appcenter_distribute_update_track_before_start_title" tools:ignore="MissingTranslation">Update track before start</string>
```

Comment:
This got me really confused when testing the UI. It should be renamed.
```suggestion
    <string name="appcenter_distribute_update_track_before_start_title" tools:ignore="MissingTranslation">When to update track</string>
```

Also it should be a menu (like start mode) not a check box. It was really odd seeing it in action.